mcs/tools/aprofutil/Program.cs

```diff
@@ -53,12 +56,21 @@ static string ProcessArguments (string [] args)
 				{ "filter-method=",
 					"Filter by method with regex {VALUE}",
 				  v => FilterMethod = new Regex (v) },
+				{ "skip-method=",
+					"Filter by method with regex {VALUE}",
```

Comment:
Could you make the description different than `filter-method=` this way it looks quite confusing